From: Tanu Kaskinen <tanuk@iki.fi>
Date: Wed, 6 Sep 2017 16:08:29 +0300
Subject: sink: remove priority bonus for the "internal" form factor

We're supposed to prioritize USB sound cards over PCI sound cards, but
the priority bonus for the "internal" form factor prevents this from
happening. Not all (if any) USB sound cards have the form factor
property set, whereas at least on my laptop the on-board sound card has
the form factor set to "internal".

(cherry picked from commit f30203c2d43eb16d76f25d87e1980e34a60753cf)
---
 src/pulsecore/sink.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/pulsecore/sink.c b/src/pulsecore/sink.c
index 2234289..cbca889 100644
--- a/src/pulsecore/sink.c
+++ b/src/pulsecore/sink.c
@@ -3534,8 +3534,6 @@ unsigned pa_device_init_priority(pa_proplist *p) {
             priority += 500;
         else if (pa_streq(s, "portable"))
             priority += 450;
-        else if (pa_streq(s, "internal"))
-            priority += 400;
     }
 
     if ((s = pa_proplist_gets(p, PA_PROP_DEVICE_BUS))) {
